tests/test.cpp

```diff
@@ -1972,6 +1988,8 @@ int main(int /*argc*/, const char * /*argv*/ []) {
 
   FlexBuffersTest();
 
+  EmptyTableTest();
```

Comment:
Beyond this test, there is no functionality that actually needs such an empty table. I'd rather this would be part of a bigger PR that actually does something with this functionality.

Reply:
Thanks aardappel! I tried: return flatbuffers::GetRoot<T>(&empty_table[4]); but the test will crash.. I was looking for internal document, did not find where the memory layout for table :-(

Reply:
static const uint8_t empty_table[] = { 0, 0, 0, /* table pointer points here */ 4, 0, 0, 0, 0 };
  return flatbuffers::GetRoot<T>(&empty_table[4]);

Reply:
oh, this should work, I should not use GetRoot<>, is this co-incident again?
  static const uint8_t empty_table[] = { 0, 0, 0, 0, /* table pointer points here */ 4, 0, 0, 0};
  return reinterpret_cast<const T *>(&empty_table[4]);